README.md

```diff
@@ -24,6 +24,7 @@
 | [Segment Open Fellowship](https://open.segment.com/fellowship) | Yes |
 | [RARE Technologies Student Incubator Program](https://rare-technologies.com/incubator/#details) | Yes |
 | [Open Summer of Code](http://2017.summerofcode.be/) | Yes** |
+| [Kharagpur Winter of Code](http://kwoc.kossiitkgp.in) | No |
```

Comment:
You should add it to the competitions section instead of the internships section @Subhajit135

Reply:
also your commit message isn't descriptive can you amend it to something like `Competitions: Add KWoC`